index.ts

```diff
@@ -1,24 +1,37 @@
+const prefix = "data-ph-capture-attribute-";
+
 // Learn more about plugins at: https://posthog.com/docs/plugins/build/overview
 function getMatchingAttributes(o, prefix) {
     return Object.keys(o).filter((k) =>
         k.toString().startsWith(`attr__${prefix}`)
     )
 }
+
+function convertDataAttribute(input: string): string {
+  //  the 6 extra chars are dure to the additional prefix "attr__"
+  const withoutDataPrefix = input.startsWith("attr__"+prefix) ? input.slice(prefix.length + 6) : input;
+
+  // Convert from kebab-case to snake_case
+  const snakeCase = withoutDataPrefix.replace(/-/g, "_");
```

Comment:
Maybe we could log an error if this changes the name? Probably means we missed something in our code.

Reply:
I'd have to check if error logging is possible in such Plugin code, but I agree it would make sense.

Reply:
`console.error` can be done anywhere, I'm pretty sure. And then we'd automatically get it in sentry.